src/_components/Editor/Editor.tsx

```diff
@@ -241,37 +421,52 @@ const Editor = React.memo(function EditorC({
     }, []);
 
     return (
-      <Fab
-        color='primary'
-        aria-label='save'
-        disabled={!enabled || isLoading || !changed}
-        style={{ position: 'fixed', bottom: 16, right: 16 }}
-        onClick={async () => {
-          setIsLoading(true);
-          setError('');
-          try {
-            const text = editorRef?.current?.getMarkdown() ?? 'error';
-            await editorSaveHandler(text ?? '');
-            setSuccess(true);
-          } catch (err: any) {
-            setError(err.message);
-          } finally {
-            setIsLoading(false);
-          }
-        }}
-      >
-        {isLoading ? <CircularProgress size={24} /> : <SaveIcon />}
-      </Fab>
+      <>
+        <Fab
+          color='primary'
+          aria-label='save'
+          disabled={!enabled || isLoading || !changed}
+          style={{ position: 'fixed', bottom: 16, right: 16 }}
+          onClick={async () => {
+            setIsLoading(true);
+            errorRef.current = '';
+            try {
+              const text = editorRef?.current?.getMarkdown() ?? 'error';
+              await editorSaveHandler(text ?? '');
+              setSuccess(true);
+            } catch (err: any) {
+              errorRef.current = err.message;
+            } finally {
+              setIsLoading(false);
+            }
+          }}
+        >
+          {isLoading ? <CircularProgress size={24} /> : <SaveIcon />}
+        </Fab>
+        <Snackbar
```

Comment:
removed the toast component in https://github.com/AirWalk-Digital/airview-next/pull/49 to be consistent with this :)